README.md

```diff
@@ -73,8 +73,7 @@ Filecoin can build and run on most Linux and MacOS systems with at least 8GB of
 Clone the git repository:
```

Comment:
I believe there are still several places in this file where we mention operations involving the GOPATH e.g. `cg ${GOPATH}`. Is it within scope of this PR to changes those commands?

Reply:
Since this README aims at developers, we might be better off leaving this recommendation alone and setting up a standard Go dev env.